modules/data-factory/variables.tf

```
@@ -0,0 +1,100 @@
+variable "name" {
+  description = "The Data Factory name."
+  type        = string
+}
+
+variable "identity_id" {
+  description = "The idetity used for Data Factory."
+  type        = string
+}
+
+variable "location" {

```

Why is "location" and no "region"?
```suggestion
variable "region" {
```